cLIMS/wetLab/forms.py

```diff
@@ -84,7 +84,7 @@ class IndividualForm(ModelForm):
         #add_related_field_wrapper(self, 'documents')
     class Meta:
         model = Individual
-        exclude = ('individual_fields','userOwner','dcic_alias','update_dcic','temp_var')
+        exclude = ('individual_fields','userOwner','dcic_alias','update_dcic')
```

Comment:
@Snehal1894 what's up with `temp_var` what was it ?

Reply:
While trying to work with fixing the multiple tabs issue, I added this temporary hidden field. But we didn't use this approach. So I forgot to delete this field from the model. It is not needed.